libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -929,31 +963,9 @@ public void logout(Account account, Activity frontActivity, final boolean showLo
     private void removeAccount(ClientManager clientMgr, final boolean showLoginPage,
     		String refreshToken, String loginServer,
     		Account account, Activity frontActivity) {
-    	cleanUp(frontActivity, account, showLoginPage);
 
-    	/*
-    	 * Removes the existing account, if any. 'account == null' does not
-    	 * guarantee that there are no accounts to remove. In the 'Forgot Passcode'
-    	 * flow there could be accounts to remove, but we don't have them, since
-    	 * we don't have the passcode hash to decrypt them. Hence, we query
-    	 * AccountManager directly here and remove the accounts for the case
-    	 * where 'account == null'. If AccountManager doesn't have accounts
-    	 * either, then there's nothing to do.
-    	 */
```

Comment:
I believe this code block has not been needed since we stopped using passcode as entropy.